modules/xmcp/gitIntegration/mdmimpl/WorkspaceObjectManagementImpl/clicommands/removexmomobject.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+ * Copyright 2023 Xyna GmbH, Germany
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+-->
+<XynaCommandLineCommand>
+  <CommandDefinition Name="removexmomobject" Description="Removes the registered XMOM object with specified fqName in the given workspace. In particular, a potentially existing deployed state is removed." />
```

Comment:
"from the given workspace", not "in the given workspace". and i don't understand the second sentence. do you refer to the deregistering from the deploymentitemstatemanagement?

Reply:
I rewrote the description and removed the second sentence. It should indicate that in addition to deregistration, the xmls files are deleted as well. Not only from rev_X/saved/XMOM but rev_X/XMOM als well. However, I think the expectation already includes not leaving the deployed xml.